frontend/tests/Message.test.tsx

```diff
@@ -3,83 +3,216 @@ import renderer from 'react-test-renderer';
 import { expect } from '@jest/globals';
```

Comment:
The upper test changes are mostly just formatting changes because I put them into describe blocks.